config/environments/test.rb

```diff
@@ -38,9 +38,6 @@
   config.static_cache_control = "public, max-age=3600"
 
   config.active_job.queue_adapter = :test
-
-  # Raise exception on mass assignment protection for Active Record models
-  config.active_record.mass_assignment_sanitizer = :strict
```

Comment:
same comment re `config.action_controller.action_on_unpermitted_parameters = :raise` in `config/environments/development.rb`

Reply:
There are 143 errors in rspec. This can be due to legacy code or old rspec tests that pass everything. It should probably be a separate ticket to do that.